### PR TITLE
test patch snapshot

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
This is meant primarily as a way to test the patch release facility.

CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

**Comment by garyverhaegen-da** [comment]
Note: I've included the 1.0 release notes because:
1. They were of course not included in this branch yet, as they were not part of the target commit for 1.0.0.
2. I wanted to have a visible change in the release notes page of the docs, so I can test my changes to the docs cron.
3. It sort of makes sense that 1.0.1 would have the notes of 1.0.0 in its release notes page.

File: LATEST [hunk 1]
```diff
@@ -1 +1 @@
-1ddcd3c0963ac5660edfd66b979e296058ea5c47 0.13.56-snapshot.20200408.3877.0.1ddcd3c0
+55fcc1647c86df5bd05f523ff7f83537b81a8b83 0.13.56-snapshot.20200416.3906.0.55fcc164
```
Comment (cocreature) [inline LATEST:1]: I don’t think it really matters but maybe `1.0.1-snapshot` would make more sense?

Comment (garyverhaegen-da) [reply LATEST:1]: Ah, good catch. Thanks.

**Comment by garyverhaegen-da** [comment]
```
Evaluating: and(succeeded(), eq(dependencies['check_for_release']['outputs']['out.is_release'], 'true'), or(eq(variables['Build.SourceBranchName'], 'master'), startsWith(variables['Build.SourceBranchName'], 'release/')))
Expanded: and(True, eq('true', 'true'), or(eq('1.0.x', 'master'), startsWith('1.0.x', 'release/')))
```
😢

**Comment by cocreature** [comment]
I’m very confused, what makes azure think it can strip away the `release/` prefix?

**Comment by garyverhaegen-da** [comment]
Apparently the fact that it ends with `/`. It's actually [documented behaviour](https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml), now that I look at it more closely.

**Pull request merged**